### PR TITLE
Ss.manage filters interaction design

### PR DESCRIPTION
**Pull request by swetha571**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

Manage Filter flyout interaction design. 
- Added new input styles.
- Font sizes are changed.

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/flyout/manageFilterFlyout.scss [hunk 1]
```diff
@@ -40,174 +62,215 @@
...
-  .Select-placeholder { color: #454a4e; }
+      &:hover,
+      &:focus { border-color: $lightestGray !important; }
+    }
 
-  .Select-arrow-zone {
-    top: 3px;
-    vertical-align: center;
-  }
+    .Select-placeholder, .Select--single > .Select-control .Select-value {
```
Comment (spryor) [inline src/components/flyout/manageFilterFlyout.scss]: separate declarations by newlines

File: src/components/flyout/manageFiltersFlyout.js [hunk 2]
```diff
@@ -224,11 +222,14 @@ class ManageFiltersFlyout extends React.Component {
...
-        <div>
+        <div className="editable-filters-wrapper">
           <label>
-            <div className="label-names">
-              {lang.FILTERNAME}
-            </div>
+          {!this.state.showCreateFilter
+            ? <div className="label-names">{lang.FILTERNAME}</div>
+            : <div className="new-filter-container">
+                <img src={Add} alt={`${Add}`} className="newfilter-add-icon" />
```
Comment (spryor) [inline src/components/flyout/manageFiltersFlyout.js]: `alt` issue as mentioned below

File: src/components/flyout/manageFiltersFlyout.js [hunk 3]
```diff
@@ -486,20 +485,17 @@ class ManageFiltersFlyout extends React.Component {
     const newGroupObj = deviceGroups[0];
     return (
       <div className="manage-filter-container">
-        <div onClick={() => this.setState({ showCreateFilter: true })} className="create-filter">
-          <img src={Add} alt={`${Add}`} className="add-icon" />
-          {lang.CREATEFILTER}
-        </div>
+        {!showCreateFilter ?
+          <div onClick={() => this.setState({ showCreateFilter: true })} className="create-filter groupname-icons">
+            <img src={Add} alt={`${Add}`} className="create-filter-icon add-icon" />
```
Comment (spryor) [inline src/components/flyout/manageFiltersFlyout.js]: `alt` issue

File: src/components/flyout/manageFiltersFlyout.js [hunk 4]
```diff
@@ -260,6 +261,16 @@ class ManageFiltersFlyout extends React.Component {
           {(group.Conditions || []).map((cond, idx) => {
             return (
               <div key={idx}>
+                {!!idx && <button
```
Comment (spryor) [inline src/components/flyout/manageFiltersFlyout.js]: multi line code block styling

File: src/components/flyout/manageFiltersFlyout.js [hunk 5]
```diff
@@ -486,20 +485,17 @@ class ManageFiltersFlyout extends React.Component {
     const newGroupObj = deviceGroups[0];
     return (
       <div className="manage-filter-container">
-        <div onClick={() => this.setState({ showCreateFilter: true })} className="create-filter">
-          <img src={Add} alt={`${Add}`} className="add-icon" />
-          {lang.CREATEFILTER}
-        </div>
+        {!showCreateFilter ?
```
Comment (spryor) [inline src/components/flyout/manageFiltersFlyout.js]: multi line code block styling

File: src/index.scss [hunk 6]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 @import 'styles/fonts/font-face-imports';
+@import 'styles//variables/colors';
```
Comment (spryor) [inline src/index.scss]: There are two `//`

File: src/index.scss [hunk 7]
```diff
@@ -14,7 +15,13 @@ body {
...
-textarea { outline: none; }
+input,
+textarea,
+button { outline: none; }
 
 #root { height: 100%; }
+
+input {
+  background-color: $gridTileDark;
+  border: 1px solid #454a4e;
```
Comment (spryor) [inline src/index.scss]: is `#454a4e` a defined color variable or a one off?

File: src/components/flyout/flyout.scss [hunk 8]
```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 $flyoutHeight: 48px;
+$FlyoutTitleFont: 16px;
```
Comment (spryor) [inline src/components/flyout/flyout.scss]: variable names should be camelCase and not start with a cap letter

File: src/index.scss (hunk 7 above)
Comment (swetha571) [reply src/index.scss]: It is not defined in a color variable file.

File: src/components/flyout/manageFilterFlyout.scss [hunk 9]
```diff
@@ -40,174 +62,216 @@
...
-  margin: -7px 3px 12px 6px;
-}
+  .editable-filters {
+    font-size: 10px !important;
+    background-color: $gridTileDark;
+    padding: 5px 0px 32px 20px;
+    margin: 16px 1px 8px 1px;
+
+    .editable-filters-wrapper {
+      border-bottom: 1.5px solid $bgBlack;
```
Comment (spryor) [inline src/components/flyout/manageFilterFlyout.scss]: No fractional px values.

File: src/components/flyout/manageFilterFlyout.scss [hunk 10]
```diff
@@ -40,174 +62,216 @@
...
-.add-condition {
-  border: none;
-  background-color: #1b1d1f;
-  display: inline-block;
-  margin-top: 8px;
-}
+  .add-condition,
+  .remove-condition {
+    border: none;
+    border-top: 1.5px solid $bgBlack !important;
```
Comment (spryor) [inline src/components/flyout/manageFilterFlyout.scss]: Fractional px value.

File: src/components/flyout/manageFiltersFlyout.js [hunk 11]
```diff
@@ -260,6 +263,19 @@ class ManageFiltersFlyout extends React.Component {
...
+                {
+                  !!idx &&
+                  <button
+                    onClick={() => {
+                      group.Conditions.splice(idx, 1);
+                      this.setEditingState(group.Id, { formChanged: true });
+                    }}
+                    type="button"
+                    className="remove-condition">
+                    <img src={CancelX} alt="CancelX" className="cancel-icon" />
```
Comment (spryor) [inline src/components/flyout/manageFiltersFlyout.js]: alt isn't useful, needs a better string

File: src/components/flyout/manageFiltersFlyout.js [hunk 12]
```diff
@@ -435,7 +439,7 @@ class ManageFiltersFlyout extends React.Component {
               }}
               className="cancel-button"
             >
-              <img src={CancelX} alt={`${CancelX}`} className="cancel-icon" />
+              <img src={CancelX} alt="CancelX" className="cancel-icon" />
```
Comment (spryor) [inline src/components/flyout/manageFiltersFlyout.js]: alt needs a better string

**Pull request merged**